frontend/src/components/Pages/FrontPage/RobotCards/RobotStatusChip.tsx

```diff
@@ -25,6 +25,10 @@ export function RobotStatusChip({ status }: StatusProps) {
             chipColor = StatusColors.Busy
             break
         }
+        case RobotStatus.Stuck: {
+            chipColor = StatusColors.Busy
```

Comment:
We should have a dedicated statuscolor for Stuck. This color can be the same as the one for busy

Reply:
```suggestion
            chipColor = StatusColors.Stuck
```

Agree with Øystein. Make this change and then add a StatusColors.Stuck that has the same value as busy.

Reply:
Done.